resources/openshift/templates/jboss64-image-streams.json

```diff
@@ -0,0 +1,152 @@
+{
+    "kind": "List",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "jboss-image-streams",
```

Comment:
the file name "jboss64-image-streams.json" doesn't make sense, it should be perhaps "jboss-datavirt64-image-streams.json"?

Reply:
FWIW, we're using datavirt63-image-stream.json upstream, https://github.com/jboss-openshift/application-templates/blob/master/datavirt/datavirt63-image-stream.json.